lib/matplotlib/tests/test_pyplot.py

```diff
@@ -335,3 +337,14 @@ def test_fallback_position():
     axtest = plt.axes([0.2, 0.2, 0.5, 0.5], position=[0.1, 0.1, 0.8, 0.8])
     np.testing.assert_allclose(axtest.bbox.get_points(),
                                axref.bbox.get_points())
+
+
+def test_doc_pyplot_summary():
+    """Test that pypot_summary lists all the plot functions."""
+    pyplot_docs = Path(__file__).parent / '../../../doc/api/pyplot_summary.rst'
+    assert pyplot_docs.exists()  # TODO: maybe skip
```

Comment:
Definitely needs a skip, since tests _can_ be run while installed, without docs anywhere near this spot. I think we do something similar for `pyplot.py`.